website/content/en/preview/troubleshooting.md

```diff
@@ -24,6 +24,18 @@ aws ssm start-session --target $INSTANCE_ID
 # Check Kubelet logs
 sudo journalctl -u kubelet
 ```
+## CoreDNS issues deploying Karpenter on Fargate
+
+Karpenter deployments on Fargate can fail if the latest CoreDSN is not used.
```

Comment:
```suggestion
Karpenter deployments on Fargate can fail if CoreDNS has nowhere to run.  It can be configured to run on either Fargate itself or in a managed node group.
```

Reply:
Good suggestion. Done.